proto/toit/model/organization.proto

```diff
@@ -18,4 +18,5 @@ message User {
     bytes organization_id = 2;
     string email = 3;
     string name = 4;
+    uint64 hubspot_id = 5;
```

Comment:
We shouldn't leak this!

Reply:
ah, okay, will delete the PR and make it internal only